### PR TITLE
Update YAML: Add unknown response value in verify location service

### PR DESCRIPTION
**Pull request by VijayKesharwani**
Adding "unknown" #19

### CODE REVIEW

File: code/API_definitions/location.yaml [hunk 1]
```diff
@@ -182,8 +182,12 @@ components:
       maximum: 200
       example: 50
     VerificationResult:
-      description: Result of a verification request, true on match
-      type: boolean
+      description: Result of a verification request, true on match, false on unmatch and unknown on no result
```
Comment (jlurien) [inline code/API_definitions/location.yaml:185]:
```suggestion
      description: Result of a verification request: TRUE on match, FALSE on no match, and UNKNOWN on location not known
```

File: code/API_definitions/location.yaml [hunk 2]
```diff
@@ -182,8 +182,12 @@ components:
...
+      type: string
+      enum:
+        - true
```
Comment (jlurien) [inline code/API_definitions/location.yaml:188]:
```suggestion
        - TRUE
```

File: code/API_definitions/location.yaml [hunk 3]
```diff
@@ -182,8 +182,12 @@ components:
...
+        - false
```
Comment (jlurien) [inline code/API_definitions/location.yaml:189]:
```suggestion
        - FALSE
```

File: code/API_definitions/location.yaml [hunk 4]
```diff
@@ -182,8 +182,12 @@ components:
...
+        - unknown
```
Comment (jlurien) [inline code/API_definitions/location.yaml:190]:
```suggestion
        - UNKNOWN
```

**Review by jlurien — COMMENTED** [review]
There are no explicit guidelines in Commonalities for the format of enum values, but almost all CAMARA APIs are formatting enum values in UPPER_CASE

**Comment by jlurien** [comment]
Also, if we commit to main branch we have to upgrade API version to 0.2.0-wip. Otherwise we can commit to dev-v0.2.0 branch. There is no clear recommendation for this across all WGs

**Comment by VijayKesharwani** [comment]
Closing this pull request because as a review comment we have to create PR to dev-v0.2.0 branch which has been done in the below PR.
[dev-v0.2.0 branch.](https://github.com/camaraproject/DeviceLocation/pull/22)

**Pull request closed without merging**